bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -10,18 +10,33 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 
-{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
-{%- block page_title -%}
-  {% if LANG == 'en-CA' %}
+
+{% if switch('firefox_new_title_experiment') %}
```

Comment:
It would be nice to include a comment here linking to the issue, as a clue for what this all means to future adventurers.